middleware/proxy/proxy.go

```diff
@@ -24,34 +24,37 @@ func Balancer(config Config) fiber.Handler {
 	cfg := configDefault(config)
 
 	// Load balanced client
-	var lbc fasthttp.LBClient
-	// Set timeout
-	lbc.Timeout = cfg.Timeout
-
-	// Scheme must be provided, falls back to http
-	// TODO add https support
-	for _, server := range cfg.Servers {
-		if !strings.HasPrefix(server, "http") {
-			server = "http://" + server
-		}
+	var lbc = &fasthttp.LBClient{}
+	if config.Client == nil {
+		// Set timeout
+		lbc.Timeout = cfg.Timeout
+		// Scheme must be provided, falls back to http
+		for _, server := range cfg.Servers {
```

Comment:
we should make clear in the readme or readme configuration that if the clients are set, then the server addresses and read and write buffer settings are not processed

Reply:
done